main.css

```diff
@@ -94,15 +94,56 @@ footer
     background-color: #061428;
 }
 
-#sidebar
+#mobileMenuIcon
 {
     position: fixed;
+    top: 35px;
+    left: 5px;
+    width: 48px;
+    height: 48px;
+    z-index: 9999;
+    background: url("assets/mobile-ui/mobilemenu.png");
```

Comment:
We might want to use fontawesome (http://fontawesome.io/icons/) for this. Some overhead, but we might need them for other parts of the UI. Maybe we can just pull them as we need them, or point this URL to a remote endpoint